schemas/http.yml

```diff
@@ -14,10 +14,7 @@
       short: HTTP request method.
       description: >
         HTTP request method.
-
-        The field value must be normalized to lowercase for querying. See
-        the documentation section "Implementing ECS".
-      example: get, post, put
+      example: get, POST, Put
```

Comment:
I consider this to be a breaking change. 

I think we should put something back that says that we used to recommend lowercasing this field, but we're removing this guidance for now, as it's a lossy normalization on a field where it's worth doing anomaly detection. Original case will be mandated as of 2.0.